packages/manager/apps/web/client/app/domain/webhosting/order/steps/domain-webhosting-order-steps.controller.js

```diff
@@ -8,8 +14,12 @@ export default class {
     this.validatingCheckout = true;
     return this.validateCheckout(this.cartId, {
       autoPayWithPreferredPaymentMethod: this.autoPayWithPreferredPaymentMethod,
-    }).catch(() =>
-      this.alertCheckoutError('domain_webhosting_order_payment_checkout_error'),
-    );
+    }).catch(() => {
+      return this.displayErrorMessage(
```

Comment:
This is weird, looks like ESLint or Prettier wasn't triggered on this file

Reply:
Rerun `yarn run lint:js` and no diff was created